changelog.d/833.feature

```diff
@@ -0,0 +1 @@
+Adds `allowedIpRanges` widgets config
```

Comment:
I realise this might be slightly verbose, but just wanted to make it clear what this does.

```suggestion
Adds widgets config (`allowedIpRanges`) option to control which IPs are allowed to access widgets.
```